### PR TITLE
Added FAQ.

### PR DESCRIPTION
**Pull request by bravegnu**

### CODE REVIEW

**Comment by bravegnu** [comment]
There is a navigation bar change, that needs to be propagated to the blog.

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+**Q. I am a student who is graduating this year before the conference
+date. Which ticket should I buy?**
+
+Students should show their valid ID card on the day of the conference
+during registration to be eligible for the student ticket. So you
+should buy the regular ticket if you won't have your ID during the
+conference.
+
+## Conctact Us
```
Comment (astronomersiva) [inline faq.md]: *Contact*

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+Please cancel the ticket and re-register. You will just incur the
+charges of the ticketing platform.
+
+**Q. Would there be any spot registration at the venue?**
+
+No, there won't be any spot registrations available at the conference
+venue.
+
+**Q. Can I switch between the workshop in session time?**
```
Comment (astronomersiva) [inline faq.md]: between workshops

File: faq.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+Yes. Workshop tickets are separate from conference tickets.
+
+**Q. There is a change in my plan so I can't attend the
+conference. Can I get a refund?**
+
+Conference tickets cannot be cancelled, but instead can be
+transferred. The procedure to do so, will be updated shortly.
+
+**Q. Is accomodation included in the ticket?**
```
Comment (astronomersiva) [inline faq.md]: accommodation

**Comment by astronomersiva** [comment]
Screen Shot 2019-05-30 at 07 44 22

@bravegnu, can you rebase your branch against master? Navbar related fixes where merged but I am seeing slight variations of them in this PR. Just want to confirm.

**Pull request merged**